packages/server-core/src/networking/instance-provision/instance-provision.class.ts

```diff
@@ -57,7 +57,6 @@ import getLocalServerIp from '../../util/get-local-server-ip'
 const releaseRegex = /^([a-zA-Z0-9]+)-/
 
 const isNameRegex = /instanceserver-([a-zA-Z0-9]{5}-[a-zA-Z0-9]{5})/
-const pressureThresholdPercent = 0.8
 
 /**
  * Gets an instanceserver that is not in use or reserved
```

Comment:
For ease maybe just add an example value of what `provisionConstraints` could be in comments section.